plugin-api/src/test/java/org/sonar/api/ce/measure/test/TestMeasureComputerContextTest.java

```diff
@@ -197,6 +198,7 @@ public void get_issues() {
       .setResolution(org.sonar.api.issue.Issue.RESOLUTION_FIXED)
       .setEffort(Duration.create(10L))
       .setType(RuleType.BUG)
+      .setCharacteristic(RuleCharacteristic.ROBUST)
```

Comment:
this change is optional, right?

Reply:
Yes, we could live without it. It doesn't hurt I think.